Fix -fpermissive error: cast GetNativeDisplay() to the type it's expecting.

--- ui/gl/gl_surface_egl.cc.orig	2013-10-03 10:24:02.101296370 +0200
+++ ui/gl/gl_surface_egl.cc	2013-10-03 10:24:17.093238232 +0200
@@ -282,7 +282,7 @@ EGLConfig NativeViewGLSurfaceEGL::GetCon
     // Get a config compatible with the window
     DCHECK(window_);
     XWindowAttributes win_attribs;
-    if (!XGetWindowAttributes(GetNativeDisplay(), window_, &win_attribs)) {
+    if (!XGetWindowAttributes((Display *)GetNativeDisplay(), window_, &win_attribs)) {
       return NULL;
     }
 
